trinity/sync/common/types.py

```diff
@@ -0,0 +1,13 @@
+from eth_typing import (
+    BlockNumber,
+)
+
+
+class SyncProgress:
```

Comment:
Can you update this to use a `typing.NamedTuple`

https://docs.python.org/3/library/typing.html#typing.NamedTuple

To make updating the `current_block` easy you could do something like:

```python
class SyncProgress(NamedTuple):
    ...

    def update_current_block(self, new_current_block):
        return type(self)(self.starting_block, new_current_block, self.highest_block)
```